### PR TITLE
Update Houston config map to properly label pods launched by the Kubernetes Pod Operator

### PR DESCRIPTION
**Pull request by rbankston**
## Description

Updated houston-configmap local airflow settings to allow kubernetes pod operator pods to be labeled properly

## 🎟 Issue(s)

Resolves astronomer/issues#2822

## 🧪  Testing

Tested with this change on my local cluster to verify that pods launched by the Kubernetes Pod Operator pods are tagged

### CODE REVIEW

File: charts/astronomer/templates/houston/houston-configmap.yaml [hunk 1]
```diff
@@ -134,32 +132,71 @@ data:
...
+                      if not pod.args:
+                          pod.args = []
+                      pod.args = pod.cmds + pod.args
                   pod.cmds = ["tini", "--", "/entrypoint"]
               else:
-                  extra_labels["kubernetes-pod-operator"] = "True"
                   # In the case of KPO, we allow Airflow to overwrite the entrypoint
                   # because we do not know what the container will be (and it's probably
-                  # not a container vendored by Astronomer, and it's definitely not
+                  # not a container vendored by Astronomer, and it's definately not
```
Comment (dimberman) [inline charts/astronomer/templates/houston/houston-configmap.yaml:160]: Why did you change this?

File: charts/astronomer/templates/houston/houston-configmap.yaml [hunk 2]
```diff
@@ -134,32 +132,71 @@ data:
...
+              from airflow.contrib.kubernetes.pod import Pod
               extra_labels = {
-                  "kubernetes-executor": "False",
                   "kubernetes-pod-operator": "False"
               }
-              if 'airflow-worker' in pod.labels.keys() or \
+              if 'airflow-worker' in pod.labels.keys() and \
                       conf.get('core', 'EXECUTOR') == "KubernetesExecutor":
-                  extra_labels["kubernetes-executor"] = "True"
+                  # extra_labels["kubernetes-executor"] = "True"
```
Comment (dimberman) [inline charts/astronomer/templates/houston/houston-configmap.yaml:144]: Why did you comment this out?

File: charts/astronomer/templates/houston/houston-configmap.yaml (hunk 1 above)
Comment (dimberman) [inline charts/astronomer/templates/houston/houston-configmap.yaml:160]:
```suggestion
                  # not a container vendored by Astronomer, and it's definitely not
```

**Review by dimberman — CHANGES_REQUESTED** [review]

File: charts/astronomer/templates/houston/houston-configmap.yaml (hunk 2 above)
Comment (wolfier) [reply charts/astronomer/templates/houston/houston-configmap.yaml:144]: This is commented out in cloud.

Comment (rbankston) [reply charts/astronomer/templates/houston/houston-configmap.yaml:144]: Yup this is the configmap in use in cloud at the moment minus some pod label affinity items

**Comment by vishwas-astro** [comment]
@rbankston where is this PR at? @dimberman have you completed your review of it?

**Review by danielhoherd — APPROVED** [review]
I like this try except approach.

File: charts/astronomer/templates/houston/houston-configmap.yaml (hunk 2 above)
Comment (dimberman) [reply charts/astronomer/templates/houston/houston-configmap.yaml:144]: Ok sounds good

**Review by dimberman — APPROVED** [review]
